controlplane/kubeadm/controllers/kubeadm_control_plane_controller.go

```diff
@@ -165,15 +166,12 @@ func (r *KubeadmControlPlaneReconciler) reconcile(ctx context.Context, kcp *cont
 	}
 
 	// TODO: handle proper adoption of Machines
-	ownedMachines, err := r.getOwnedMachines(
-		ctx,
-		kcp,
-		types.NamespacedName{Namespace: cluster.Namespace, Name: cluster.Name},
-	)
+	allMachines, err := r.getMachines(ctx, types.NamespacedName{Namespace: cluster.Namespace, Name: cluster.Name})
 	if err != nil {
-		logger.Error(err, "failed to get list of owned machines")
+		logger.Error(err, "Failed to get list of machines")
```

Comment:
Nit: I think we're lower casing all messages, but we haven't finalised that logging format, so not a blocker.

Reply:
Got it. I see Uppercase throughout right now, so if it's ok we can leave this and do a change to lowercase for the entire package when we finalize the format.

Reply:
@randomvariable For Info/Error logs the messages should be capitalized, but for error strings:

```go
errors.New("this is an error string")
fmt.Errorf("this is also an error string: %v", err)
```

those should be lower case. This is the convention we've followed, but you're right in that we haven't formalized it.

Reply:
Uppercase log messages, lowercase errors then? That's follows community convention, I think.

Reply:
> Uppercase log messages, lowercase errors then

This is what we've been following